website/src/app/blog/sans-io/readme.mdx

```diff
@@ -453,15 +453,15 @@ event loop are: `handle_timeout`, `handle_input`, `poll_transmit` and
 protocols can be implemented with these or some variation of them. As a result,
 it is very easy to compose these state machines together: want to query 5 STUN
 servers for your public IP? No problem. Just make 5 `StunBinding`s and call them
-in order[^3].
```

Comment:
FYI: In Markdown, these don't have to be ordered, they just have to be unique. They don't even have to be numbers I think!